doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -835,6 +840,10 @@ Release notes for 0.5.0 and 0.5.1 can be found in :file:`ncs/nrf/modules/lib/zcb
 Trusted Firmware-M
 ==================
 
+* Added:
+
+  * Added support for a PSA attestation key using an identity key stored in the Key Management Unit (KMU).
```

Comment:
```suggestion
  * Added support for the PSA Initial Attestation Token (IAT)
  * Added support for provisioning an identity key that is used by the IAT
```

Is "PSA attestation key" accurate? Should we say we support the IAT instead?

Is it too bold to say that we support provisioning devices now?

Reply:
"PSA attestation key using an identity key"

Is the identity key not the PSA attestation key ?

Reply:
Done